cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java

```diff
@@ -137,24 +139,14 @@ void testApplyAddRemove() throws Throwable {
 			ContentCategoryBulkAction action = (ContentCategoryBulkAction) this.getAction();
 			this.checkItems(contentIds, action.getSelectedIds());
 			this.checkItems(categoryCodes, action.getCategoryCodes());
-			String commandId = action.getCommandId();
-			assertNotNull(commandId);
-			
-			result = this.executeCheckResult(currentUser, commandId);
-			assertEquals(Action.SUCCESS, result);
-			
-			result = this.executeViewResult(currentUser, commandId);
-			assertEquals(Action.SUCCESS, result);
-
-			commandId = ((ContentBulkAction) this.getAction()).getCommandId();
-			this.checkReport(commandId, size, size, size, 0, ApsCommandStatus.COMPLETED);
 
+//			result = this.executeCheckResult(currentUser);
+//			assertEquals(Action.SUCCESS, result);
+//			result = this.executeViewResult(currentUser);
+//			assertEquals(Action.SUCCESS, result);
```

Comment:
Can we remove these commented lines?

Reply:
there was a comment in the original code in entando-ps, i will remove it

Reply:
done